src/debug/NodeDebugProvider.ts

```diff
@@ -15,7 +15,8 @@ export const nodeDebugConfig: DebugConfiguration = {
     type: 'node',
     request: 'attach',
     port: defaultNodeDebugPort,
-    preLaunchTask: hostStartTaskName
+    preLaunchTask: hostStartTaskName,
+    restart: true
```

Comment:
As discussed offline, "restart" doesn't seem to work on Mac. We'll need to figure out why that is before we can merge